### ISSUE TITLE
Accessibility: WinForms other controls should support UIA Notification event except those already supported controls: Label, LinkLabel and GroupBox

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#4494]
* .Net Core Version:
.Net SDK 6.0.100-preview.1.21071.10
* Have you experienced this same bug with .NET Framework?:
Yes
## Issue description:
So far, Label, LinkLabel, GroupBox controls have supported raising notification event, want to extend support for raising notifications event to other controls

To raise the notification event on a control, the RaiseAutomationNotification method must be called on the control’s accessible object. E.g: label1.AccessibilityObject.RaiseAutomationNotification(AutomationNotificationKind.Other,  AutomationNotificationProcessing.All,  “Hi, I am Narrator. This is a label.”);

Here is a winforms sample app for label, linklabel, groupBox controls, and test result:
[WinFormsApp129.zip](https://github.com/dotnet/winforms/files/5871369/WinFormsApp129.zip)
![image](https://user-images.githubusercontent.com/26474449/105804698-6c09f300-5fdb-11eb-81ae-8d61d76e52b8.png)

## Expected result
More Winforms controls support UIA notification event

## Actual result
Just Label, LinkLabel, GroupBox support UIA notification event.

**Comment by Danil-Andrianov** [comment]
@Olina-Zhang 
Hi, Olina! The issue was fixed in [this PR](https://github.com/dotnet/winforms/pull/2096)
Additionally, it's also OK in .NET Core 3.1
Attach archive with application.
[Issue4494.zip](https://github.com/dotnet/winforms/files/7453077/Issue4494.zip)
Please, check the gif files below. First six controls support UIA provider by itself and remaining ones don't.
- **Support UIA provider**
<details>
<summary> Button</summary>

![button](https://user-images.githubusercontent.com/74228865/139664481-091c4c20-a35e-4b52-bb18-638fac09f9ef.gif)
</details>
<details>
<summary> GroupBox</summary>

![groupBox](https://user-images.githubusercontent.com/74228865/139665575-0ae7da53-13ed-43b0-be1c-2fe47c64ae74.gif)
</details>
<details>
<summary> TabControl</summary>

![tabControl](https://user-images.githubusercontent.com/74228865/139665629-7bc9659a-cf09-4847-b7d7-a6f26310a0f8.gif)
</details>
<details>
<summary> Label</summary>

![label](https://user-images.githubusercontent.com/74228865/139665691-79a6a57c-c2cb-4104-a7fb-7e10fa2e9548.gif)
</details>
<details>
<summary> PropertyGrid</summary>

![propertyGrid](https://user-images.githubusercontent.com/74228865/139666125-57e2211c-dbfc-49e0-ab60-78d02e8c2e1b.gif)
</details>
<details>
<summary> PictureBox</summary>

![pictureBox](https://user-images.githubusercontent.com/74228865/139666162-35476909-7f79-41d6-8460-4c7eb57782be.gif)
</details>

- **Don't support UIA provider**
<details>
<summary> TextBox</summary>

![textBox](https://user-images.githubusercontent.com/74228865/139667385-6cddf2c5-52b1-4611-ba3d-b5d034220d0e.gif)
</details>
<details>
<summary> TreeView</summary>

![treeView](https://user-images.githubusercontent.com/74228865/139667416-390048ac-b066-444c-8523-15c772e770ab.gif)
</details>
<details>
<summary> DateTimePicker</summary>

![dateTimePicker](https://user-images.githubusercontent.com/74228865/139667455-32e49bb8-d62a-4023-b2b4-388140df6c69.gif)
</details>
<details>
<summary> FlowLayoutPanel</summary>

![flowLayoutPanel](https://user-images.githubusercontent.com/74228865/139667489-604d402d-000b-497e-82d7-54e373c0ba6b.gif)
</details>
<details>
<summary> RichTextBox</summary>

![richTextBox](https://user-images.githubusercontent.com/74228865/139667519-30c100f1-616a-4fdb-8124-073503834029.gif)
</details>
<details>
<summary> TableLayoutPanel</summary>

![tableLayoutPanel](https://user-images.githubusercontent.com/74228865/139667561-fc22a1a1-a324-4c3e-9221-687924855b95.gif)
</details>

**Comment by Olina-Zhang** [comment]
@Danil-Andrianov Thanks for your testing, I have the same testing result with you.

**Comment by Danil-Andrianov** [comment]
@Olina-Zhang The problem is not reproduced, therefore, I agree. Please, close this issue, I just can't do this because of lack of rights.
